firefox-ios/Client/Frontend/Library/Bookmarks/BookmarksViewController.swift

```diff
@@ -149,10 +149,27 @@ class BookmarksViewController: SiteTableViewController,
                     self?.flashRow()
                 }
                 self?.updateEmptyState()
+                self?.updateParentViewControllerTitle()
             }
         }
     }
 
+    private func updateParentViewControllerTitle() {
+        guard viewModel.isBookmarkRefactorEnabled else { return }
```

Comment:
`BookmarksViewController` is only used if the bookmarks refactor is enabled, making this check redundant